CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.17.0-dev
```

Comment:
Technically `1.16.1-dev` would be better (there are no new features, its just a patch release).

Reply:
Makes sense, updated.